.gitignore

```diff
@@ -8,3 +8,4 @@ RUNNING_PID
 .bsp/
 .DS_Store
 node_modules
+bundle.js
```

Comment:
question (non-blocking): thoughts on building the JS bundle somewhere other than the root directory? e.g. `build/`?

Reply:
It's not actually being built into the root dir. It's being built into `public/javascripts/bundle.js` (see the `path` field of `webpack.config.js`). But for whatever reason, putting the full path here ("public/javascripts/bundle.js) didn't successfully ignore the file. I can try a couple other styles of syntax, but just "bundle.js" does the trick regardless of location.

Reply:
Ah cool, no need to change then